src/main/java/com/qubit/solution/fenixedu/integration/cgd/servlet/CgdintegrationInitializer.java

```diff
@@ -48,6 +51,9 @@ public Object call() {
                 if (!dynamicGroup.isDefined()) {
                     dynamicGroup.toPersistentGroup();
                 }
+                
+                new WebServiceClientConfiguration(CgdStudentPhotoClient.class.getName());
```

Comment:
@pmmsa isn't this creating a new domain object each time the application is starting?
I think we should first test that such client does not exist.